rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -722,6 +861,13 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 					continue; // system files are not included in the file list
 				}
 
+				const bool is_media = entry.name == "ICON0.PNG" || entry.name == "ICON1.PAM" || entry.name == "PIC1.PNG" || entry.name == "SND0.AT3";
+
+				if (is_media && !i || !is_media && i)
```

Comment:
Warnings about parens. It looks like you wanted this:
```suggestion
				if ((is_media && !i) || (!is_media && i))
```

Reply:
Thx. But this is most likely one of the lines that's gonna be replaced anyway :)